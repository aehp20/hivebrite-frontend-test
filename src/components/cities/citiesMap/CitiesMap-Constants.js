export const DEFAULT_ZOOM = 13
export const DEFAULT_MARKER = {
  "city": "Paris",
  "growth_from_2000_to_2013": "4.8%", 
  "latitude": 48.864716, 
  "longitude": 2.349014, 
  "population": "60405840", 
  "rank": "1", 
  "state": "ÃŽle de France"
}
