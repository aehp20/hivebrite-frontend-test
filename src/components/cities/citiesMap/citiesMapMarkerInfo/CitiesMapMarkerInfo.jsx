import React from 'react'

export function CitiesMapMarkerInfo(props) {
  const {marker} = props

  return (
    <span>
      City: {marker.city}<br/>
      Growth from 2000 to 2013: {marker.growth_from_2000_to_2013}<br/>
      Population: {marker.population}<br/>
      Rank: {marker.rank}<br/>
      State: {marker.state}
    </span>
  )
}
