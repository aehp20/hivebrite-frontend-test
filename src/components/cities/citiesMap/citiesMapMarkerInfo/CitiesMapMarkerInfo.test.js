import React from 'react'
import renderer from 'react-test-renderer'

import {DEFAULT_MARKER} from '../CitiesMap-Constants'
import {CitiesMapMarkerInfo} from './CitiesMapMarkerInfo'

test('CitiesMapMarkerInfo test', () => {
  const marker = DEFAULT_MARKER
  const component = renderer.create(
    <CitiesMapMarkerInfo  marker={marker} />,
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
