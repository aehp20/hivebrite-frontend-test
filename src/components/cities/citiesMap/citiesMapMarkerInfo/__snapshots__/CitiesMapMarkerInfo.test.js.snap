// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesMapMarkerInfo test 1`] = `
<span>
  City: 
  Paris
  <br />
  Growth from 2000 to 2013: 
  4.8%
  <br />
  Population: 
  60405840
  <br />
  Rank: 
  1
  <br />
  State: 
  ÃŽle de France
</span>
`;
